src/cmd/nsfs.js

```diff
@@ -235,6 +239,30 @@ async function init_nsfs_system(config_root) {
         throw error;
     }
 }
+// Create an anonymous account for anonymous request. Use this account UID and GID for bucket access.
+async function add_anonymous_account() {
```

Comment:
@naveenpaul1 @guymguym 
I had a quick sync with Guy today, and we also discussed this account creation. Guy suggests that for now we won't create this account automatically, instead we will - 
1. Let the user create an anonymous account using the CLI (uid/gid will be determined on the create/update request, no access keys are attached to this account)
2. Add a documentation section for anonymous and explain that before running an anonymous s3 request there are steps to take - 
- Create an account using the CLI while specifying --anonymous flag
- Create bucket policy on a bucket specifying that anonymous access is allowed
- Run the s3 anonymous request on the bucket

This solution makes our life a bit easier, but we need to pay attention to the fact that currently we can not have multiple anonymous users. An easy way to restrict this is to use a hard-coded name as you did. WDYT?

Reply:
changes are commited, Please review